compiler/rustc_data_structures/src/sync.rs

```diff
@@ -99,11 +59,6 @@ cfg_if! {
         }
 
         impl<T: Copy> Atomic<T> {
-            #[inline]
-            pub fn into_inner(self) -> T {
-                self.0.into_inner()
```

Comment:
Can we keep this such that the non-parallel and parallel versions export the same api.